cpp/kiss_icp/3rdparty/tbb/tbb.cmake

```diff
@@ -43,6 +43,6 @@ set(TBB_ROOT ${INSTALL_DIR} CACHE INTERNAL "TBB_ROOT Install directory")
 add_library(TBBHelper INTERFACE)
 add_dependencies(TBBHelper external_tbb)
 target_include_directories(TBBHelper SYSTEM INTERFACE ${INSTALL_DIR}/include)
-target_link_directories(TBBHelper INTERFACE ${CMAKE_INSTALL_LIBDIR})
+target_link_directories(TBBHelper INTERFACE ${INSTALL_DIR}/lib ${INSTALL_DIR}/lib64)
```

Comment:
This was actually the propper way of doing it. Some stuff was missing though

1. `include(GNUInstallDirs)`
1. `${INSTALL_DIR}/${CMAKE_INSTALL_LIBDIR}`

`CMAKE_INSTALL_LIBDIR` only spits `lib` or `lib64`. We need to preppend our custom, local direcotry, `INSTALL_DIR`